src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/multi-cluster/multi-cluster-form/multi-cluster-form.component.html

```diff
@@ -188,6 +188,24 @@
                   i18n>This field is required.</span>
           </div>
         </div>
+        <div class="form-group row"
+             *ngIf="!remoteClusterForm.getValue('showToken') && action !== 'edit'">
+          <label class="cd-col-form-label"
+                 for="ttl"
+                 i18n>Login Expiration</label>
+          <div class="cd-col-form-input">
+            <select class="form-select"
+                    id="ttl"
+                    formControlName="ttl"
+                    name="ttl">
+              <option value="1">1 day</option>
+              <option value="7">1 week</option>
+              <option value="15"
+                      [selected]="true">15 days</option>
+              <option value="30">1 month</option>
```

Comment:
Should we say it 30 days ?
just thinking since 1 month could be - 31, 30 just to be precise or its fine.

Reply:
Thanks @afreen23 , changed it to 30 days